MyWeight/Screens/List/MassView.swift

```diff
@@ -68,10 +72,21 @@ public class MassView: UIView, ViewModelOwner {
 
         dateLabel.numberOfLines = 0
         dateLabel.textAlignment = .right
+        
+        contentView.addSubview(line)
+        
+        line.translatesAutoresizingMaskIntoConstraints = false
+        
+        line.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: grid*4).isActive = true
+        line.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -grid*4).isActive = true
+        line.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: 0).isActive = true
+       
+        line.widthAnchor.constraint(equalToConstant: 100).isActive = true
```

Comment:
Since you have `leading` and `trailing` constraints the `width` might be necessary. Unless you want to have a maximum width.
Makes sense?